composer.json

```diff
@@ -25,32 +25,34 @@
     "alchemy/zippy": "^0.4.4"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.7"
+    "phpunit/phpunit": "^5.7",
+    "joomla-projects/joomla-testing": "dev-container-test",
+    "joomla-projects/joomla-testing-robo": "dev-container-test"
   },
   "scripts": {
     "test": [
       "rm -rf dockyard/*",
       "phpunit --testdox"
     ],
     "build": [
-      "rm -rf dockyard/*",
-      "phpunit tests/DockerComposeGeneratorTest",
-      "cd dockyard && docker-compose build",
+      "rm -rf dockyard_testing/*",
+      "phpunit tests/TestingDockerComposeGeneratorTest",
+      "cd dockyard_testing && docker-compose build",
       "cd .."
     ],
     "start": [
-      "cd dockyard && docker-compose up -d",
+      "cd dockyard_testing && docker-compose up -d",
       "cd .."
     ],
     "stop": [
-      "cd dockyard && docker-compose stop",
+      "cd dockyard_testing && docker-compose stop",
       "cd .."
     ],
     "clean": [
       "@stop",
-      "cd dockyard && docker-compose down",
+      "cd dockyard_testing && docker-compose down",
       "cd ..",
-      "rm -rf dockyard/*"
+      "rm -rf dockyard_testing/*"
     ]
   }
 }
```

Comment:
Why did you rename `dockyard`? It is already for testing only. The composer commands are ment to support development of the Virtualisation tool, for explorative testing of the generated setup.